ensenso_camera/src/camera.cpp

```diff
@@ -652,8 +652,11 @@ geometry_msgs::TransformStamped Camera::stampedLinkToCamera()
 
 tf2::Transform Camera::getCameraToLinkTransform()
 {
-  // The NxLib will always give the transfrom from the camera to the link target in camera coordinates.
+  // The NxLib will always give the transform from the camera to the link target in camera coordinates.
   tf2::Transform transform;
+  // Always initialize transform otherwise the transform will be invalid
+  transform.setIdentity();
```

Comment:
We could simplify this to:

`tf2::Transform transform = tf2::Transform::getIdentity();`

Reply:
Good point I updated the code.